docs/reference/aggregations/metrics/top-metrics-aggregation.asciidoc

```diff
@@ -71,6 +72,7 @@ request. So,
 `metrics` selects the fields of the "top" document to return. You can request
 a single metric with something like `"metric": {"field": "m"}` or multiple
 metrics by requesting a list of metrics like `"metric": [{"field": "m"}, {"field": "i"}`.
+The fields can be <<numbers,number>>, <<keywords,keyword>>, or <<ips,ip>>.
```

Comment:
These xrefs are breaking the docs CI. This should fix them up.

```suggestion
The fields can be <<number,numeric>>, <<keyword,keyword>>, or <<ip,`ip`>>.
```

Reply:
:+1: